actions/lib/action.py

```diff
@@ -104,3 +109,31 @@ def _run_client_method(self, method, method_kwargs, format_func, format_kwargs=N
         result = method(**method_kwargs)
         result = format_func(result, **format_kwargs or {})
         return result
+
+    def _manipulate_rule(self, name, pack, enabled):
+        rule_name = '{}.{}'.format(pack, name)
+        failure_reason = None
+        rule = None
+        try:
+            rule = self.client.rules.get_by_name(name=name, pack=pack)
+            if not rule:
+                failure_reason = 'rule not found'
+        except Exception as exc:
+            failure_reason = exc
+
+        if failure_reason:
+            return 'Could not get rule {}: {}'.format(rule_name, failure_reason)
+
+        rule_enabled = rule.enabled
+
+        if isinstance(rule_enabled, bool) and rule_enabled == enabled:
```

Comment:
is there a case where this could not be bool?

Reply:
Oh, I think it's leftovers from the initial implementation where it was possible :) Then I changed it, but missed this line. Let me double check and fix if needed.

Reply:
Changed that.